### DISCUSSION TITLE
Edit metadata - select another page for thumbnail

### DISCUSSION THREAD

**Discussion by HaveClassWillCode** [gotson/komga#1726] [Ideas]
Some magazine PDFs have an advertisement as page 1.
It would be great if you could edit the metadata, and point to page 2 or 3 to get the proper magazine cover, which is then used for the thumbnail.

**Comment by gotson** [comment]
You can already do that, open the book with the reader, and use the 3-dots menu and select"Set page as poster for book"

<img width="299" alt="image" src="https://github.com/user-attachments/assets/d0d81ea6-0740-4001-be5c-0350d2ed2d26">

**Reply by HaveClassWillCode** [reply]
Thanks!
I was utterly confused as I didn't see those icons.
I'm new to Komga, but had read the docs, and did click around, so was utterly confused.
By accident I now found that I have to click on the page to see these icons...
